.maintain/Dockerfile

```diff
@@ -1,59 +1,57 @@
-# Note: We don't use Alpine and its packaged Rust/Cargo because they're too often out of date,
-# preventing them from being used to build Substrate/Polkadot.
+FROM  rust:1.46.0-alpine3.12 as builder
```

Comment:
Should we use 1.47.0 now?

Reply:
I have not tried to build rc5 with 1.47, but will update if it does. 

fyi. currently v.149 nightly is broken for substrate v2.0.0, so will keep at 1.48 nightly